lib/network_utils.ml

```diff
@@ -905,6 +905,27 @@ module Proc = struct
 end
 
 module Ovs = struct
+  let match_multiple patterns s =
```

Comment:
I think it would be good to compile the patterns once when the module is initialised:

```
let patterns = List.map Re.Perl.compile_pat
  [ "no bridge named (.*)\n"
  ; "no row \"(.*)\" in table Bridge"
  ]
```

Afterwards, only use `patterns` for matching.